### ISSUE TITLE
Image missing a text alternative: Archivo branding image

### ISSUE THREAD

**Issue by elynema** [avalonmediasystem/avalon#5343]
Sites identified: Archivo
Page(s) identified: all pages

WCAG compliance level: A

Description:

All non-decorative images must have a text alternative (also known as "alt text").

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBb3hnIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--dc0cc5e8ab42cdc6cb5ae039e58ceb1655d9b0cb/image.png)

Note: does this image count as a decorative image that does not need alt text?

# Done Looks Like
- [x] empty alt text attribute added for the logo image (indicating decorative image)

**Comment by joncameron** [comment]
Is there a way to mark an image as decorative so it doesn't get flagged?

**Comment by joncameron** [comment]
Deployed 11/5 and empty alt text now appears in the markup for the image.
